src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Enum getRandomColor() {
+        Random random = new Random();
```

Comment:
Please make Random class level variable

Reply:
Still not resolved

Reply:
Not fixed. Do you need to create a new instance of `Random` every time you call a method `getRandomColor`? `Random` should be a class-level variable.